handsontable/types/plugins/dialog/dialog.d.ts

```diff
@@ -0,0 +1,27 @@
+import Core from '../../core';
+import { BasePlugin } from '../base';
+
+export interface DialogConfig {
+  content?: string | HTMLElement;
+  customClassName?: string;
+  background?: 'solid' | 'semi-transparent';
+  contentBackground?: boolean;
+  contentDirections?: 'row' | 'row-reverse' | 'column' | 'column-reverse';
+  animation?: boolean;
+  closable?: boolean;
+}
+
+export type Settings = boolean | DialogConfig;
+
+export class Dialog extends BasePlugin {
+  static PLUGIN_KEY: string;
+  static PLUGIN_PRIORITY: number;
+  static DEFAULT_CONFIG: DialogConfig;
+
+  constructor(hotInstance: Core);
+  isEnabled(): boolean;
```

Comment:
The methods is already defined in the BaseClass
```suggestion
  constructor(hotInstance: Core);
```